CHANGELOG.md

```diff
@@ -10,8 +10,13 @@
 
 - Box [Breaking]: Removed support for `shape` prop values `roundedTop`, `roundedRight`, `roundedBottom`, and `roundedLeft` (#657)
 - Touchable [Breaking]: Removed support for `shape` prop values `roundedTop`, `roundedRight`, `roundedBottom`, and `roundedLeft` (#657)
-- Run codemod for breaking changes: `cd gestalt
+- Box [Breaking]: Replace `shape` prop with `rounding` (#666)
```

Comment:
@jennyscript Feel free to combine those: `Box / Mask / Touchable: Replace `shape` prop with `rounding` (#666)`